connector_prestashop/models/account_invoice/importer.py

```diff
@@ -0,0 +1,244 @@
+        if fpos:
+            fpos_obj = self.env['account.fiscal.position']
+            account_id = fpos_obj.map_account(
+                self.session.cr,
+                self.session.uid,
+                fpos,
+                account_id
+            )
+        return {
+            'quantity': 1,
+            'product_id': product.id,
+            'name': order_line['name'],
+            'invoice_line_tax_id': [(6, 0, order_line['tax_id'])],
+            'price_unit': price_unit,
+            'discount': order_line['discount'],
+            'account_id': account_id,
+        }
+
+    def _get_shipping_order_line(self, record):
+        binder = self.binder_for('prestashop.sale.order')
+        sale_order = binder.to_internal(record['id_order'], unwrap=True)
+        if not sale_order.carrier_id:
+            return None
+        sale_order_line_ids = self.env['sale.order.line'].search([
+            ('order_id', '=', sale_order.id),
+            ('product_id', '=', sale_order.carrier_id.product_id.id),
+        ])
+        if not sale_order_line_ids:
+            return None
+        return sale_order_line_ids[0].read([])
+
+    def _invoice_line(self, record, fpos):
+        order_line = self._get_order_line(record['id_order_detail'])
+        tax_ids = []
+        if order_line is None:
+            product_id = None
+            name = "Order line not found"
+            account_id = None
+        else:
+            product = order_line.product_id
+            product_id = product.id
+            name = order_line.name
+            for tax in order_line.tax_id:
+                tax_ids.append(tax.id)
+            account_id = product.property_account_income_id.id
+            if not account_id:
+                categ = product.categ_id
+                account_id = categ.property_account_income_categ_id.id
+        if fpos and account_id:
+            fpos_obj = self.session.pool['account.fiscal.position']
+            account_id = fpos_obj.map_account(
+                self.session.cr,
+                self.session.uid,
+                fpos,
+                account_id
+            )
+        if record['product_quantity'] == '0':
+            quantity = 1
+        else:
+            quantity = record['product_quantity']
+
+        if self.backend_record.taxes_included:
+            price_unit = record['amount_tax_incl']
+        else:
+            price_unit = record['amount_tax_excl']
+
+        try:
+            price_unit = float(price_unit) / float(quantity)
+        except ValueError:
+            pass
+
+        discount = False
+        if price_unit in ['0.00', ''] and order_line is not None:
+            price_unit = order_line['price_unit']
+            discount = order_line['discount']
+        return {
+            'quantity': quantity,
+            'product_id': product_id,
+            'name': name,
+            'invoice_line_tax_id': [(6, 0, tax_ids)],
```

Comment:
Same comment than before
```suggestion
            'invoice_line_tax_ids': [(6, 0, tax_ids)],
```